css/home.css

```diff
@@ -1 +1,190 @@
 @import "/css/nav.css";
+
+.slider-container {
+  position: relative;
+  width: 100%;
+  height: 50rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  animation: slider 9s infinite linear;
+}
+
+@keyframes slider {
+  0% {
+    background-image: url("/images/amanda-vick-zw_oaDbfzyE-unsplash.jpg");
+  }
+  35% {
+    background-image: url("/images/ashim-d-silva-ZmgJiztRHXE-unsplash.jpg");
+  }
+  75% {
+    background-image: url("/images/junko-nakase-Q-72wa9-7Dg-unsplash.jpg");
+  }
+  100% {
+    background-image: url("/images/sarah-dorweiler-fr0J5-GIVyg-unsplash.jpg");
+  }
+}
+
+.shop-btn {
+  position: absolute;
+  padding: 2rem 4rem;
+  font-size: 2rem;
+  cursor: pointer;
+  box-shadow: 2px 2 px var(--nav-secondary-color);
+  border: none;
+  font-weight: 500;
+  transition: 0.2s ease;
+  margin-top: 30rem;
+}
+
+.shop-btn:hover {
+  background-color: var(--black);
+  color: var(--nav-text-color);
+}
+
+.featured-pages-heading {
+  text-align: center;
+  font-size: 3rem;
+  margin: 2rem;
+}
+
+.card-container {
+  width: 100%;
+}
+
+.cards {
+  display: flex;
```

Comment:
You can put the display , align , justify in a common class

Reply:
thanks I'll refactor